### PR TITLE
Make `impeller/aiks/...` compatible with `.clang-tidy`.

### PR DESCRIPTION
**Pull request by matanlurey**

### CODE REVIEW

File: impeller/aiks/canvas_recorder.h [hunk 1]
```diff
@@ -16,6 +16,7 @@ namespace impeller {
 /// golden tests can be written at a higher level, migrate these to
 /// flutter::DisplayListOpType.
 enum CanvasRecorderOp : uint16_t {
+  // NOLINTBEGIN(readability-identifier-naming)
```
Comment (gaaclarke) [inline impeller/aiks/canvas_recorder.h:19]: You don't just want to rename them?

**Review by gaaclarke — APPROVED** [review]
lgtm

File: impeller/aiks/canvas_recorder.h (hunk 1 above)
Comment (matanlurey) [reply impeller/aiks/canvas_recorder.h:19]: I could. It caused problems with a macro used to map to something else, but maybe I was just tired at the time. I can give it a second shot.

Comment (matanlurey) [reply impeller/aiks/canvas_recorder.h:19]: Ok I was just being lazy, this seems fine :)

**Pull request merged**